app/lib/munkireport/Database.php

```diff
@@ -98,4 +98,27 @@ public function get_driver()
     {
         return $this->getDBH()->getAttribute(\PDO::ATTR_DRIVER_NAME);
     }
+    
+	/**
+     * Get database size
+     *
+     * @return number
+     **/
+    public function get_totalsize()
+    {
+    
+    	$sql = 'SELECT sum(round(((DATA_LENGTH + INDEX_LENGTH) / 1024 / 1024 ), 2)) from information_schema.TABLES';
```

Comment:
This only works for MySQL and I think it also needs a WHERE TABLE_SCHEMA = 'insert munkireport table from config'